sys-apps/ignition/ignition-9999.ebuild

```diff
@@ -40,11 +40,6 @@ RDEPEND="
 
 RDEPEND+="${DEPEND}"
 
-PATCHES=(
-	"${FILESDIR}/0001-providers-allow-FetchConfig-to-mutate-the-fetcher.patch"
```

Comment:
I guess we should remove these two files from git as well, right?

Reply:
Good catch, done